ROADMAP.md

```diff
@@ -0,0 +1,14 @@
+# Skopeo Roadmap
+
+The Skopeo development team reviews feature requests from its various stakeholders for consideration
+quarterly.  Skopeo maintainers then prioritize these features.   Top features are then assigned to
+one or more engineers.
+
+
+## Future feature focus
+
+* OCI artifact support
+* Integration of composefs
+* Partial pull support (zstd:chunked).
+* Performance and stability improvements.
+* Reductions to the size of the Skopeo binary.
```

Comment:
Do we say anything about `skopeo sync` here? It’s the primary non-c/image code in Skopeo.

“`skopeo sync` remains a feature in our lineup” 😈 
Maybe:
> `skopeo sync` exists, and bugs in it should be fixed, but we don’t have much of an ambition to compete with much larger projects like https://github.com/openshift/oc-mirror .

… or maybe nothing? I weakly think contributors should be told. Or maybe we could attract new contributors/maintaienrs of this? We’d need to think through what that would mean. (@baude )